Assets/Scripts/Player/Transformation.cs

```diff
@@ -77,6 +79,11 @@ public class Transformation : MonoBehaviour {
 		particleEffect.Stop();
 	}
 
+	private void Update() {
+		if (unableTransform)
+			RecheckCrouched();
```

Comment:
This function does more than just check something so the name is quite misleading. Maybe just inline it?